components/02-elements/list/_list.scss

```diff
@@ -43,6 +46,31 @@ $list-divider: 1px solid $gray-dark !default;
             padding: $list-padding--horizontal;
         }
     }
+    &--desktop-hotizontal {
+        @include mq($screen-m) {
+            display: flex;
+            flex-direction: row;
+            flex-wrap: nowrap;
+        }
+        .list__item {
+            border-bottom: $list-divider;
+            padding: $list-padding--horizontal;
+            &:first-child {
+                @include mq($screen-m) {
+                    border-top: 0;
+                }
+            }
+            &:last-child {
+                border-bottom: none;
+            }
+            @include mq($screen-m) {
+                border-bottom: 0;
+            }
+        }
+        .list__link {
+            display: block;
+        }
+    }
```

Comment:
@talalus  You mixed up here a `horizontal` with `divided` + custom code, that's not correct aproach. Remember that components needs to be reusable and highli customizable, not created just to use it in one place.
I already have rewritten this component in develop branch, so just revert these changes and adjust your code in the theme.